renovate.json

```diff
@@ -27,8 +27,17 @@
       "updateTypes": ["major", "minor"],
       "rangeStrategy": "bump"
     },
+    {
+      "baseBranchList": ["master"],
+      "labels": [":package: master"]
+    },
+    {
+      "baseBranchList": ["v13.x"],
```

Comment:
Let's merge it for now. Once we have the branch creation implemented in the release-bot, we can update the base branches for renovate.